content/_data/changelogs/lts.yml

```diff
@@ -2001,6 +2001,42 @@
     pull: 3591
     message: >
       A configured quiet period was interpreted as milliseconds, instead of seconds. (Regression in 2.82)
+- version: "2.138.2"
+  date: 2018-10-10
+  changes:
+  - type: security
+    message: Important security fixes.
+    references:
+    - url: https://jenkins.io/security/advisory/2018-10-10/
+      title: security advisory
+  - type: major bug
+    references:
+    - issue: 53239
+    - issue: 52804
+    - issue: 51136
+    - issue: 52358
+    pull: 3602
+    message: >
+      Update Winstone-Jetty from 4.4 to 5.0 to fix HTTP/2 support and threading problems on hosts with 30+ cores.
+  - type: rfe
+    pull: 3604
+    references:
+    - url: https://github.com/jenkinsci/jep/tree/master/jep/214
+      title: JEP-214
+    message: >
+      Add extensibility mechanism for anonymous usage statistics with initial implementation collecting information about applied security fix escape hatches.
```

Comment:
Do we want to announce the extensibility provided it is practically unusable for plugin authors or users (since it is restricted)?

Reply:
+1, extensibility is not what we backport